src/itest/kotlin/nl/lifely/zac/itest/ZaakRestServiceTest.kt

```diff
@@ -106,7 +107,7 @@ class ZaakRestServiceTest : BehaviorSpec({
                       "isProcesGestuurd": false,
                       "isVerlengd": false,
                       "kenmerken": [],
-                      "omschrijving": "dummyOmschrijving",
+                      "omschrijving": "$ZAAK_DESCRIPTION_2",
```

Comment:
There's no need to put a string `ZAAK_DESCRIPTION_2` in a string through parameter injection. Or am I missing something here? And this is done in a number of places, I'd expect a detekt warning would've mentioned this, no?